src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -218,6 +226,13 @@ export function transform(formConfig, form) {
       ),
     );
 
+  const addBackRatedDisabilities = formData =>
```

Comment:
All of this code & related cleanup code is going into separate transformers instead of `setActionTypes` because `setActionTypes` is exported and tested separately from the `transform` function, so doesn't have access to the saved rated disabilities we just set up above.